package.json

```diff
@@ -12,11 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "dotenv -- webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "start-mac": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "validate-init": "python _validate_init.py",
         "prepublishOnly": "npm run validate-init",
-        "build:js": "webpack --mode production",
+        "build:js": "dotenv -- webpack --mode production",

```

Comment:
What is the "--" for here? Should it be "--webpack"?

Reply:
It seems we do need like this  as we are using dotenv-cli - the purpose of the -- is to separate the options intended for dotenv-cli from the subsequent command we want to run (in this case, webpack)

Reply:
Ah, interesting! Thank you for explaining